### DISCUSSION TITLE
视频预合成支持生成HEVC格式么？

### DISCUSSION THREAD

**Discussion by dugufly** [Tencent/libpag#1627] [Q&A] [answered]
实测都是AVC的格式

**Comment by kevingpqi123** [comment, marked as answer]
这里需要的背景是啥，BMP 预合成中的视频序列帧编码采用的是 h.264, 这里的主要考量是编码格式支持的广泛度以及解码速度，HEVC 的支持广泛度不如 h.264

**Reply by dugufly** [reply]
H265文件 的体积小一些吧？

**Reply by dugufly** [reply]
如果基于ffmpeg做h265的编解码，兼容性也没有问题吧？

**Reply by kevingpqi123** [reply]
你说的这些我们都清楚，H265文件是会更小，但解码的计算量更大哈，会更耗时，视频解码优先考虑的是硬解，软解仅仅是回退
这里的编码格式更寻求的是一种普适性更强、支持范围更广的格式
